lib/routes/cline/blog.ts

```diff
@@ -7,56 +7,110 @@ const rootUrl = 'https://cline.bot';
-            const author = metaEl.find('span').first().text().trim();
-            const dateStr = metaEl.find('span').eq(2).text().trim();
-            const pubDate = dateStr ? parseDate(dateStr) : undefined;
+    // Archive page structure - articles in chronological list
+    $('div.group > div.space-y-4 > div.group').each((_, article) => {
+        const element = $(article);
 
-            const summary = element.find('.text-slate-600').text().trim();
-            const imgSrc = element.find('img').attr('src') || '';
+        // Title and link
+        const titleEl = element.find('h3.text-lg.font-semibold.text-gray-900 a, a.text-gray-900');
+        const title = titleEl.text().trim();
+        const link = titleEl.attr('href') || element.find('a[href*="/blog/"]').attr('href');
+        const fullLink = link ? (link.startsWith('http') ? link : `${rootUrl}${link.startsWith('/') ? link : `/${link}`}`) : '';
 
-            return {
+        // Date extraction from format like "Aug 12, 2024"
+        const dateEl = element.find('div.text-sm.text-gray-500');
+        const dateStr = dateEl.text().trim();
+        const pubDate = dateStr ? parseDate(dateStr) : undefined;
+
+        // Description
+        const summary = element.find('p.text-gray-600').text().trim();
+
+        if (title && link) {
+            articles.push({
                 title,
                 link: fullLink,
                 pubDate,
-                author,
-                description: `<img src="${imgSrc}" /><p>${summary}</p>`,
-            };
-        })
-        .filter((item) => item.title && item.link);
+                author: 'Cline Team',
+                description: `<p>${summary}</p>`,
+            });
+        }
+    });
+
+    // Fallback: try main blog page structure if archive doesn't work
+    if (articles.length === 0) {
+        // Main blog page structure
+        $('article.group').each((_, article) => {
+            const element = $(article);
+
+            const title = element.find('h2, h3').text().trim();
+            const link = element.find('a').first().attr('href');
+            const fullLink = link ? (link.startsWith('http') ? link : `${rootUrl}${link.startsWith('/') ? link : `/${link}`}`) : '';
+
+            // Extract date and author
+            const metaText = element.find('.text-sm.text-slate-500, .text-xs.text-slate-500').text().trim();
+            const dateMatch = metaText.match(/(?:\w+\s*•\s*)?([A-Za-z]+\s+\d{1,2},?\s+\d{4})/);
+            const pubDate = dateMatch ? parseDate(dateMatch[1]) : undefined;
+
+            const authorMatch = metaText.match(/^([^•]+)•/);
+            const author = authorMatch ? authorMatch[1].trim() : 'Cline Team';
+
+            const summary = element.find('p.text-slate-600, p.text-slate-700').text().trim();
+            const imgSrc = element.find('img').attr('src') || '';
+
+            if (title && link) {
+                articles.push({
```

Comment:
Use [map](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Array/map) instead of [push](https://www.measurethat.net/Benchmarks/Show/11532/0/push-vs-map)